composer.lock

```diff
@@ -517,16 +517,16 @@
         },
         {
             "name": "consolidation/self-update",
-            "version": "2.0.3",
+            "version": "2.0.4",
```

Comment:
It's a good idea to change our requirement in composer.json to ^2.0.4, since we have a hard dependency on that version now.